CMakeLists.txt

```diff
@@ -127,6 +127,7 @@ else()
 endif()
 
 option(EOSIO_ENABLE_DEVELOPER_OPTIONS "enable developer options for EOSIO" OFF)
+option(EOSIO_REQUIRE_FULL_VALIDATION "remove runtime options allowing light validation" OFF)
```

Comment:
Update PR description to indicate this PR pulls in `EOSIO_REQUIRE_FULL_VALIDATION` option.